tools/run_tests/xds_k8s_test_driver/tests/url_map/retry_test.py

```diff
@@ -66,8 +67,10 @@ def _build_retry_route_rule(retryConditions, num_retries):
 class TestRetryUpTo3AttemptsAndFail(xds_url_map_testcase.XdsUrlMapTestCase):
 
     @staticmethod
-    def supported_clients() -> Tuple[str]:
-        return 'cpp', 'java', 'go'
+    def is_supported(config: TestConfig) -> bool:
+        if config.client_lang in ['cpp', 'java', 'go']:
+            return config.version_ge('v1.40.x')
```

Comment:
Retry should be the same as affinity. go version >= 1.41.x

Reply:
Done.